examples/eql/harmonic_spherical.py

```diff
@@ -0,0 +1,121 @@
+"""
+Gridding in Spherical coordinates
+=================================
+
+When gridding magnetic or gravity data on large regions thought the equivalent
+layer technique, the curvature of the Earth must be taken into account.
+Projecting data for large regions may introduce projection errors on the
+predictions.
+
+:class:`harmonica.EQLHarmonicSpherical` is able to estimate the
+coefficients of a set of point sources (the equivalent layer) that fit the
+observed harmonic data given in Spherical coordinates.
+It allows to benefit from the same advantages that the equivalent layer
+technique offers while taking into account the curvature of the Earth.
+"""
+import numpy as np
+import boule as bl
+import verde as vd
+import harmonica as hm
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
```

Comment:
```suggestion
import numpy as np
import cartopy.crs as ccrs
import matplotlib.pyplot as plt
import boule as bl
import verde as vd
import harmonica as hm
```

I like to put our packages last to call more attention to them :slightly_smiling_face:

Reply:
Ok, agree. I tend to list packages in increasing line length order, it just please my eyes haha.